packages/flutter/test/material/slider_test.dart

```diff
@@ -352,6 +352,45 @@ void main() {
     expect(SchedulerBinding.instance.transientCallbackCount, equals(0));
   });
 
+  testWidgets('Should not throw exception when dragged after Slider is disposed', (WidgetTester tester) async {
```

Comment:
This test pass even without the change, is this testing a real issue?

Reply:
@liumcse 
It seems that the test passes without the fix on the master.